neptune/new/internal/init_impl.py

```diff
@@ -84,6 +85,96 @@ def init(project: Optional[str] = None,
          monitoring_namespace: str = "monitoring",
          flush_period: float = 5,
          proxies: dict = None) -> Run:
+    """Starts a new tracked run, and append it to the top of the Runs table view.
+
+    Args:
+        project(str, optional): Name of a project in a form of `namespace/project_name`. Defaults to `None`.
+            If `None`, the value of `NEPTUNE_PROJECT` environment variable will be taken.
+        api_token(str, optional): User’s API token. Defaults to `None`.
+            If `None`, the value of `NEPTUNE_API_TOKEN` environment variable will be taken.
+            .. note::
+                It is strongly recommended to use `NEPTUNE_API_TOKEN` environment variable rather than placing your
+                API token in plain text in your source code.
+        run (str, optional): An existing run's identifier like 'SAN-1' in case of resuming a tracked run.
+            Defaults to `None`.
+            A run with such identifier must exist. If `None` is passed, starts a new tracked run.
+        custom_run_id (str, optional): A unique identifier to be used when running Neptune in pipelines.
+            Defaults to `None`.
+            Make sure you are using the same identifier throughout the whole pipeline execution.
+        mode (str, optional): Connection mode in which the tracking will work. Defaults to `'async'`.
+            Possible values 'async', 'sync', 'offline', and 'debug'.
+        name (str, optional): Editable name of the run. Defaults to `'Untitled'`.
+            Name is displayed in the run's Details and in runs view as a column.
+        description (str, optional): Editable description of the run `''`.
+            Description is displayed in the run's Details and can be displayed in the runs view as a column.
+        tags (list of str or str, optional): Tags of the run. Defaults to `[]`.
+            They are editable after run is created.
+            Tags are displayed in the run's Details and can be viewed in Runs table view as a column.
+        source_files (list of str or str, optional): List of source files to be uploaded.
+            Uploaded sources are displayed in the run’s Source code tab.
+            Unix style pathname pattern expansion is supported. For example, you can pass '*.py' to upload all python
+            source files from the current directory.
+            If `None` is passed, Python file from which run was created will be uploaded.
+        capture_stdout (bool, optional): Whether to send run's stdout. Defaults to `True`.
+            Tracked metadata will be stored inside monitoring_namespace.
```

Comment:
monitoring_namespace is a param so it should be with '' or `` (not sure which one is correct here, but definitely not without :D) 
Same comment two next parameters.

Reply:
+1